docs/sources/plugins/developing/development.md

```diff
@@ -28,21 +28,18 @@ Example plugins
 - [Clock panel](https://github.com/grafana/clock-panel)
 - [Pie chart panel](https://github.com/grafana/piechart-panel)
 
-There are two blog posts about authoring a plugin that might also be of interest to any plugin authors.
+There are two blog posts about authoring a plugin that might also be of interest to other users.
```

Comment:
See earlier comment about audience.

Also, is this content written for Grafana 3.0 still valid for 6.6?

Reply:
@oddlittlebird I was told we shouldn't point to old blog posts but instead to our recently released tutorials section.

Reply:
@marcusolsson , are the tutorials ready?

Reply:
@oddlittlebird https://grafana.com/tutorials/